tests/golden.xml

```diff
@@ -34,28 +34,36 @@ expected: &#34;False&#34;
     time="0.000021009"
     timestamp="2021-05-17T15:36:10.205608409Z"
     hostname="localhost"
-    tests="3" failures="0" errors="0" skipped="1">
+    tests="4" failures="1" errors="0" skipped="1">
     <properties/>
     <testcase
       file="tests/ExampleSpec.hs"
-      line="26"
+      line="28"
+      name="throws a colourful exception"
+      classname="Some section/A grouped context"
+      time="0.000012194">
+        <failure type="error">ColourfulException</failure>
+    </testcase>
+    <testcase
+      file="tests/ExampleSpec.hs"
+      line="27"
       name="gets skipped"
       classname="Some section/A grouped context"
       time="0.000006765">
-      <skipped>tests/ExampleSpec.hs:26:9
+      <skipped>tests/ExampleSpec.hs:27:9
 some reason</skipped>
     </testcase>
     <testcase
       file="tests/ExampleSpec.hs"
-      line="23"
+      line="24"
       name="also happens in a group"
       classname="Some section/A grouped context"
       time="0.000007421"/>
     <testcase
       file="tests/ExampleSpec.hs"
-      line="21"
+      line="22"
       name="happens in a group"
       classname="Some section/A grouped context"
       time="0.000006823"/>
   </testsuite>
-</testsuites>
+</testsuites>
```

Comment:
I think your editor trimmed off the final newline here. Not a big deal, just FYI in case there's a setting you want to tweak to avoid that generally.